src/js/components/AllAttributesSummaryWindow.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import Attribute from './Attribute';
+import { connect } from "react-redux"
+import "../../styles/attributes.scss";
+
+@connect((store) => {
+    return {
+        lastStep: store.nextStep
+    };
+})
+export default class AllAttributesSummaryWindow extends Component {
+    showAbilities() {
+        return Object.keys(this.props.lastStep.abilities).map((key, index, value) =>
```

Comment:
Using `index` is antipattern ;)

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318#.uxrmfcanz